sdk/communication/communication-sms/samples/typescript/src/sendSms.ts

```diff
@@ -7,38 +7,83 @@
+import { DefaultAzureCredential } from "@azure/identity";
 
 // Load the .env file if it exists
 import * as dotenv from "dotenv";
-dotenv.config();
+dotenv.config({
+  path: "sample.env"
+});
 
-// You will need to set this environment variables or edit the following values
-const connectionString =
-  process.env["COMMUNICATION_CONNECTION_STRING"] || "<communication service connection string>";
+export const createSmsClient = () => {
+  // You will need to set this environment variable or edit the following value
+  const connectionString =
+    process.env["COMMUNICATION_CONNECTION_STRING"] ||
+    "endpoint=https://<resource-name>.communication.azure.com/;<access-key>";
+  return new SmsClient(connectionString);
+};
 
-export const main = async () => {
-  console.log("== Send SMS Message Sample ==");
+export const createSmsClientWithToken = () => {
+  // You will need to set this environment variable or edit the following value
+  const endpoint =
+    process.env["COMMUNICATION_ENDPOINT"] || "https://<resource-name>.communication.azure.com";
+  //AZURE_CLIENT_SECRET, AZURE_CLIENT_ID and AZURE_TENANT_ID environment variables are needed to create a DefaultAzureCredential object.
+  return new SmsClient(endpoint, new DefaultAzureCredential());
+};
 
-  const client = new SmsClient(connectionString);
+export const sendingSmsMessage = async () => {
+  console.log("== Send SMS Message, Default Options ==");
+  const client = createSmsClient();
+  const sendResults = await client.send({
+    from: "<from-phone-number>", // Your E.164 formatted phone number used to send SMS
+    to: ["<to-phone-number-1>", "<to-phone-number-2>"], // The list of E.164 formatted phone numbers to which message is being sent
+    message: "Hello World via SMS!" // The message being sent
+  });
 
-  // Send SMS message
+  for (const sendResult of sendResults) {
+    if (sendResult.successful) {
+      console.log("Success: ", sendResult);
+    } else {
+      console.error("Something went wrong when trying to send this message: ", sendResult);
+    }
+  }
+  console.log("== Done: Send SMS Message, Default Options ==");
+};
+
+export const sendingSmsMessageWithOptions = async () => {
+  console.log("== Send SMS Message With Options ==");
+  const client = createSmsClient();
   const sendResults = await client.send(
     {
-      from: "<phone number>", // Your E.164 formatted phone number used to send SMS
-      to: ["<phone number>", "<phone number>", "<phone number>"], // The list of E.164 formatted phone numbers to which message is being send
+      from: "<from-phone-number>", // Your E.164 formatted phone number used to send SMS
+      to: ["<to-phone-number-1>", "<to-phone-number-2>"], // The list of E.164 formatted phone numbers to which message is being sent
       message: "Hello World via SMS!" // The message being sent
     },
     {
       enableDeliveryReport: true,
-      tag: "customTag"
+      tag: "TypeScriptSMSSample"
```

Comment:
In C#, the message and tag are related so that users can get an idea of how to use the tag. (e.g. Message is "Weekly Promotion", and tag is "marketing". Wondering if we should do something here as well.

Reply:
will do, @lsundaralingam can you do that for python too then?